.gitignore

```diff
@@ -1,4 +1,3 @@
 **/node_modules
 .nyc_output
-coverage
-server/node_modules
```

Comment:
Node modules should be ignored... Not sure why they're not, even with this line deleted, since we have `**/node_modules` ignored already. Either way, looks like the node modules are getting checked in again.